packages/api/src/client/http/index.ts

```diff
@@ -1,8 +1,12 @@
+import {PORT} from '../../constants.js'
 import {type BlueAsset} from '../../DataBuilder.js'
 
 export class HttpApiClient {
 	async #fetch(endpoint: string) {
-		const response = await fetch(`https://blue.leofcoin.org/${endpoint}`)
+		const response = await fetch(
+			// @ts-ignore
```

Comment:
yes... and no..

Reply:
So we need vite type to just set meta.env?